frontend/templates/product/index.tsx

```diff
@@ -34,6 +40,22 @@ import { useInternationalBuyBox } from '@templates/product/hooks/useInternationa
 export const ProductTemplate: NextPageWithLayout<ProductTemplateProps> = () => {
    const { product } = useProductTemplateProps();
    const [selectedVariant, setSelectedVariantId] = useSelectedVariant(product);
+   const apiClient = useIFixitApiClient();
+   const [reviewsData, setReviewsData] = useState<
+      ProductReviewData | null | undefined
+   >();
+   useEffect(() => {
+      if (!product.iFixitProductId) {
+         return;
+      }
```

Comment:
I think you can simplify the type to ProductReviewData | null if you init this with null.
```suggestion
   const [reviewsData, setReviewsData] = useState<
      ProductReviewData | null
   >(null);
   useEffect(() => {
      if (!product.iFixitProductId) {
         return;
      }
```